src/components/Stakeholders/Add.js

```diff
@@ -0,0 +1,354 @@
+                  <TextField
+                    error={!!errors?.last_name}
+                    helperText={errors.last_name}
+                    className={classes.textField}
+                    label="Last Name"
+                    variant="outlined"
+                    name="last_name"
+                    onChange={handleChange}
+                    value={data.last_name}
+                    onKeyDown={handleEnterPress}
+                  />
+                </FormControl>
+              </>
+            ) : (
+              <FormControl xs={12} sm={6} className={classes.root}>
+                <TextField
+                  error={!!errors?.org_name}
+                  helperText={errors.org_name}
+                  className={classes.textField}
+                  label="Organization Name"
+                  variant="outlined"
+                  name="org_name"
+                  onChange={handleChange}
+                  value={data.org_name}
+                  onKeyDown={handleEnterPress}
+                />
+              </FormControl>
+            )}
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                error={!!errors?.email}
+                helperText={errors.email}
+                className={classes.textField}
+                label="Email"
+                variant="outlined"
+                name="email"
+                onChange={handleChange}
+                value={data.email}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                error={!!errors?.phone}
+                helperText={errors.phone}
+                className={classes.textField}
+                label="Phone"
+                variant="outlined"
+                name="phone"
+                type="number"
+                onChange={handleChange}
+                value={data.phone}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Website"
+                variant="outlined"
+                name="website"
+                onChange={handleChange}
+                value={data.website}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Logo Url"
+                variant="outlined"
+                name="logo_url"
+                onChange={handleChange}
+                value={data.logo_url}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
```

Comment:
Perhaps we should try to pre-populate this based on the organization name.

Reply:
Would it exist already or would this be a way of naming it?

We should be able to query for it.

Reply:
Good point. I thought of this as a way of naming the map, but you're right that it might already exist.
This might be a good way to add new maps via the webmap API, but I'm not sure how automated this process currently is.
Leave it as it is for now and we can raise an issue to discuss options.